### ISSUE TITLE
Closing the Trust Gap: Proper Source Verification (Christian Reitwiessner, Solidity team)

### ISSUE THREAD

**Issue by ninabreznik** [playproject-io/trustlessness#6]

**Comment by ninabreznik** [comment]
What are the open issues to make it happen? Next steps? Who's working on this?

**Comment by chriseth** [comment]
There is a web interface for a prototype now: http://komputing.org:8081/

**Comment by ninabreznik** [comment]
Nice! So, if I understand correctly people can now send their contracts through the verification process here (and not to Etherscan anymore)?

**Comment by chriseth** [comment]
Yes! Unfortunately, I don't see anyone doing that, so it would be great to know if it works of whether there are major UX barriers (which I would not be surprised of).
